src/utils/Logger.ts

```diff
@@ -172,6 +172,20 @@ class Logger {
     const logFilePath = this.getReactNativeLogsFilePath()
     console.debug('React Native logs will be piped to ' + logFilePath)
 
+    // If the creation time of the file is more than 28 days ago, delete it.
+    RNFS.stat(logFilePath)
+      .then((stat) => {
+        if (+stat.ctime < +new Date() - 4 * 7 * 24 * 60 * 60 * 1000) {
```

Comment:
Are we worried about losing valuable log content that could help support fix issues?

Another simple option that avoids this is to use a different log file for each day or week (and delete the old ones).  We keep recent content, but can still clean up the old stuff that doesn't matter.

Reply:
If the user is sending a log to support in the worst case we would have from the application start to sending the contact support which would likely contain the issue; additionally, since all the transactions are on the chain support can and developers can also be triage via [Celo Explorer](https://explorer.celo.org/). 

We could create logs for each week or day and combine them when sending to support, we'd just need a bit more logic around combining logs alongside finding and deleting old logs. If this is preferred we can certainly do it either in a follow up PR or adjust this PR.

Reply:
@jeanregisser I did end up creating log files for each day and combining them; however, given the performance hit of combining several files, can take upwards of 25 seconds, maybe this is a good path to go if we don't implement https://github.com/valora-inc/wallet/issues/2701.